### PR TITLE
SOL

### PR DESCRIPTION
**Pull request by neagarr**

### CODE REVIEW

File: characters/knights.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+class Knights:
+    list_of_heroes = []
```
Comment (ArsenPidhoretskyi) [inline characters/knights.py:2]: Make it as dict
and prefix list_of_ is redundant

Comment (ArsenPidhoretskyi) [reply characters/knights.py:2]: Fix in all places

File: items/armour.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
+class Armour:
+    list_of_armour = []
```
Comment (ArsenPidhoretskyi) [inline items/armour.py:2]: Why do you need this?

File: items/potion.py [hunk 3]
```diff
@@ -0,0 +1,15 @@
+class Potion:
+    list_of_potions = []
```
Comment (ArsenPidhoretskyi) [inline items/potion.py:2]: Why do you need this?

File: items/potion.py [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+    def __init__(self, potion: dict, owner: str) -> None:
+        self.owner = owner
+        self.name = potion["name"]
+
+        if "power" in potion["effect"].keys():
+            self.power = potion["effect"]["power"]
+        if "hp" in potion["effect"].keys():
+            self.hp = potion["effect"]["hp"]
+        if "protection" in potion["effect"].keys():
+            self.protection = potion["effect"]["protection"]
```
Comment (ArsenPidhoretskyi) [inline items/potion.py:13]: Use loop and getattr and setattr

File: items/weapon.py [hunk 5]
```diff
@@ -0,0 +1,9 @@
+class Weapon:
+    list_of_weapons = []
```
Comment (ArsenPidhoretskyi) [inline items/weapon.py:2]: Why do you need this?

**Review by ArsenPidhoretskyi — CHANGES_REQUESTED** [review]
Please simplify logic it is so hard

**Comment by neagarr** [comment]
Don't know why so many other files was added to my commit

**Review by Grayder0152 — CHANGES_REQUESTED** [review]
Pls move envs files to gitignore

**Review by vsmutok — CHANGES_REQUESTED** [review]
Please remove `vent` form PR

**Comment by neagarr** [comment]
> Please remove `vent` form PR

what does mean "PR"? )

**Pull request closed without merging**